CHANGELOG.md

```diff
@@ -49,8 +49,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.9.2] - 2023-01-20
 
 ### Added
-- Adds ability to pipe queries to the CLI
-- Adds ability to run PartiQL files as executables by adding support for shebangs
+- Adds ability to pipe queries to the CLI.
+- Adds ability to run PartiQL files as executables by adding support for shebangs.
+- Adds initial syntax for CREATE TABLE.
```

Comment:
How about adding a link to an issue in https://github.com/partiql/partiql-docs and labeling the implementation as experimental? something like [this](https://github.com/partiql/partiql-lang-kotlin/pull/831/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL44-L46)